.github/workflows/ci.yml

```diff
@@ -29,10 +29,9 @@ jobs:
       - name: Install dependencies
         run:
           python -m pip install --upgrade pip
-          pip install -r requirements.txt flake8 pytest
-      - name: Lint with flake8
-        run:
-          flake8 --ignore=E501 mintapi # pep8 flake ignoring long lines
+          pip install -r requirements.txt pytest
+      - name: lint
+        uses: psf/black@stable
```

Comment:
Awesome! So will this just fail if the code isn't properly styled, or does it actually make changes to the PR?

Reply:
It just fails currently. It doesn't look like the support committing the changes made in the pipeline currently.